scripts/test-utils/processingMessageWithRuntimeUpgrade.js

```diff
@@ -4,8 +4,8 @@ const { readFileSync } = require("fs");
 const assert = require("assert/strict");
```

Comment:
could you provide separate test for code upload?
just reuse function from utils

Reply:
two random tests together seems really strange

Reply:
It's fix solution. I can add TODO. We can add separate test resolving issue #1378 .
I don't wanna to solve test problems in this PR

Reply:
> It's fast solution. I can add TODO. We can add separate test resolving issue #1378 .
> I don't wanna to solve test problems in this PR

Okay, no problem